### ISSUE TITLE
"Send to" dropdown should not have the menu role

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/orangelight#3138]
**Expected behavior**

The "Send to" dropdown (which includes SMS, Email, and Printer options) does not have the menu aria role, since it is not "[a list of menu items [...] presented in a manner similar to a menu on a desktop application](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role)".

**Actual behavior**
The "Send to" dropdown has the "menu" aria role, and it is not implemented correctly.

**Steps to replicate**

1. Go to https://wave.webaim.org
2. Enter https://catalog.princeton.edu/catalog/SCSB-7868075 as the address to check
3. On the details tab, notice that it says "1 Broken Aria Menu"
4. Press the "i" button next to the Broken Aria Menu error for more details about the error (specifically, that this role requires at least one child with the menuitem, menuitemcheckbox, or menuitemradio role).

**Impact of this bug**
According to WebAIM's quick tip in [this newsletter](https://webaim.org/newsletter/2021/january): "Defining role="menu" on navigation that is not a true ARIA menu can break screen reader behavior."  For what it's worth, I didn't notice any bad behavior in voiceover.

**Implementation Note**
1. Remove the erroneous role from the tag.

**Comment by kevinreiss** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/OrangelightBibdataRequests-571691cab409d8d821b873be/issues/pulibrary/orangelight/3138?planning-poker) with ZenHub @maxkadel @regineheberlein @rladdusaw @sandbergja
